implementations/micrometer-registry-atlas/src/main/java/io/micrometer/atlas/SpectatorToDoubleGauge.java

```diff
@@ -26,7 +26,7 @@
  * identical to com.netflix.spectator.api.ObjectGauge which is not accessible in
  * Spectator.
  */
-class SpectatorToDoubleGauge<T> extends AbstractMeter<T> implements Gauge {
+class SpectatorToDoubleGauge<T>extends AbstractMeter<T> implements Gauge {
```

Comment:
See https://github.com/spring-io/spring-javaformat/issues/363

Reply:
Oh wow. I didn't even know that's valid Java.